### DISCUSSION TITLE
Is it possible to run colour-science together with colour-demosaicing and colour-checker-detection or is there a version conflict?

### DISCUSSION THREAD

**Discussion by richardstardust** [colour-science/colour#948] [Q&A] [answered]
I am trying to use functions from colour-science and colour-demosaicing and colour-checker-detection.
I am not able to run some examples from colour-checker-detection and my guess is that there is some version conflict between the packages.
Is this correct?

**Comment by thomasmansencal** [comment, marked as answer]
Hi @richardstardust,

The updated versions of *colour-hdri* and *colour-checker-detection* have not been updated yet but should be over the weekend. *colour-demosaicing* has had one compatibility release with *colour-0.4.0* but I will do another one to ensure its `setup.py` file is working as expected.

**Reply by richardstardust** [reply]
Hi KelSolaar,
Sounds great! Looking forward to try it out!

**Reply by thomasmansencal** [reply]
They should all have been updated, let me know if any issues!

**Reply by richardstardust** [reply]
Great! My first tests have been successful.
Thank you for maintaining these very useful python packages!
